packages/local_auth/local_auth_ios/example/ios/RunnerTests/FLTLocalAuthPluginTests.m

```diff
@@ -175,6 +174,83 @@ - (void)testFailedAuthWithoutBiometrics {
                                                                 @"localizedReason" : reason,
                                                               }];
 
+  XCTestExpectation *expectation = [self expectationWithDescription:@"Result is called"];
+  [plugin handleMethodCall:call
+                    result:^(id _Nullable result) {
+                      XCTAssertTrue([NSThread isMainThread]);
+                      XCTAssertTrue([result isKindOfClass:[FlutterError class]]);
+                      [expectation fulfill];
+                    }];
+  [self waitForExpectationsWithTimeout:kTimeout handler:nil];
+}
+
+- (void)testFailedWithKnownErrorCode {
```

Comment:
I think `testFailedAuthWithBiometrics` already handles a "known error code", this is more testing `LAErrorTouchIDNotEnrolled` explicitly, right?
```suggestion
- (void)testFailedTouchIDNotEnrolled {
```

Reply:
Youre right, it's kind of a dupe - Ill remove the test